src/test/java/seedu/address/testutil/TypicalRooms.java

```diff
@@ -34,6 +36,8 @@ public static RoomList getTypicalRoomList() {
     }
 
     public static List<Room> getTypicalRooms() {
-        return new ArrayList<>(Arrays.asList(ROOM_NO_PATIENT_NO_TASK));
+        System.out.println(ROOM_PATIENT_ALICE_NO_TASK.getTaskList());
```

Comment:
remember to remove `println` statment here. you can consider adding a `//TODO` before using `println` statements, so you won't forget to remove them